symphony-utility/src/main/java/org/symphonyoss/symphony/bots/utility/client/SymphonyClientUtil.java

```diff
@@ -0,0 +1,86 @@
+package org.symphonyoss.symphony.bots.utility.client;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.symphonyoss.client.SymphonyClient;
+import org.symphonyoss.client.exceptions.MessagesException;
+import org.symphonyoss.symphony.clients.model.SymMessage;
+import org.symphonyoss.symphony.clients.model.SymStream;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Created by nick.tarsillo on 12/6/17.
+ *
+ * Helper class for retrieving Symphony data.
+ */
+public class SymphonyClientUtil {
```

Comment:
Please create unit tests for this class.

Reply:
I don't see any use for tests that only test third party functions, but done!